src/seed/seed-db.js

```diff
@@ -1,19 +1,198 @@
-import ApolloClient from 'apollo-client'
-import dotenv from 'dotenv'
-import fetch from 'node-fetch'
-import { HttpLink } from 'apollo-link-http'
-import { InMemoryCache } from 'apollo-cache-inmemory'
-import Seed from './data/index'
+import Factory, { create, relate } from './factories'
 
-dotenv.config()
+/* eslint-disable no-multi-spaces */
+(async function () {
+  try {
+    await Promise.all([
+      create('badge', { id: 'b1', key: 'indiegogo_en_racoon', type: 'crowdfunding', status: 'permanent', icon: '/img/badges/indiegogo_en_racoon.svg' }),
+      create('badge', { id: 'b2', key: 'indiegogo_en_rabbit', type: 'crowdfunding', status: 'permanent', icon: '/img/badges/indiegogo_en_rabbit.svg' }),
+      create('badge', { id: 'b3', key: 'indiegogo_en_wolf',   type: 'crowdfunding', status: 'permanent', icon: '/img/badges/indiegogo_en_wolf.svg' }),
+      create('badge', { id: 'b4', key: 'indiegogo_en_bear',   type: 'crowdfunding', status: 'permanent', icon: '/img/badges/indiegogo_en_bear.svg' }),
+      create('badge', { id: 'b5', key: 'indiegogo_en_turtle', type: 'crowdfunding', status: 'permanent', icon: '/img/badges/indiegogo_en_turtle.svg' }),
+      create('badge', { id: 'b6', key: 'indiegogo_en_rhino',  type: 'crowdfunding', status: 'permanent', icon: '/img/badges/indiegogo_en_rhino.svg' })
+    ])
 
-if (process.env.NODE_ENV === 'production') {
-  throw new Error('YOU CAN`T SEED IN PRODUCTION MODE')
-}
+    await Promise.all([
+      create('user', { id: 'u1', name: 'Peter Lustig',       role: 'admin',     email: 'admin@example.org' }),
+      create('user', { id: 'u2', name: 'Bob der Baumeister', role: 'moderator', email: 'moderator@example.org' }),
+      create('user', { id: 'u3', name: 'Jenny Rostock',      role: 'user',      email: 'user@example.org' }),
+      create('user', { id: 'u4', name: 'Tick',               role: 'user',      email: 'tick@example.org' }),
+      create('user', { id: 'u5', name: 'Trick',              role: 'user',      email: 'trick@example.org' }),
+      create('user', { id: 'u6', name: 'Track',              role: 'user',      email: 'track@example.org' }),
```

Comment:
We have to rename the users. I accidently used the German versions of Dagobert Ducks grandchildren here. But the English version is different, see here: https://en.wikipedia.org/wiki/Huey,_Dewey,_and_Louie